src/main/java/Problem3/Problem3.java

```diff
@@ -5,7 +5,13 @@
 import java.io.PrintStream;
 
 public class Problem3 {
+        // got help from geeks for geeks.com
     public static void printListInReverse(ListNode list, PrintStream print) {
-        // homework
+        if(list != null) {
```

Comment:
You don't need 2 checks (:
`if (list == null) {
            return;
        }
        printListInReverse(list.next, print);
        print.println(list.val);`